backend/src/main/java/ca/bc/gov/hlth/mohums/controller/DashboardController.java

```diff
@@ -0,0 +1,56 @@
+package ca.bc.gov.hlth.mohums.controller;
+
+import java.util.List;
+import java.util.Map;
+import java.sql.*;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class DashboardController {
+
+    @Value("${spring.datasource.url}")
+    private String url;
+
+    @Value("${spring.datasource.username}")
+    private String username;
+
+    @Value("${spring.datasource.password}")
+    private String password;
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @GetMapping("/dashboard/active-user-count")
```

Comment:
We suggest `metrics` instead of `dashboard`. A "dashboard" is where the data is used in the web app, but this is the API, so it shouldn't have any references to the UI.